src/components/index/sections/BetaPopUpDialog.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { ChangeEvent, useEffect, useState } from 'react'
+import { useTranslation } from 'next-i18next'
+import {
+  Button,
+  Grid,
+  Checkbox,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  FormControl,
+  FormControlLabel,
+  Typography,
+} from '@mui/material'
+import { CircleOutlined } from '@mui/icons-material'
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+
+import BetaPopUpTextCard from 'components/index/helpers/betaPopUp/BetaPopUpTextCard'
+import BetaManIcon from 'components/index/icons/BetaManIcon'
+import BetaWomanIcon from 'components/index/icons/BetaWomanIcon'
+
+let showAgainProp = true
+
+export default function BetaPopUpDialog() {
+  const { t } = useTranslation('index')
+  const [open, setOpen] = useState(false)
+  const [showAgain, setShowAgain] = useState(true)
+
+  const handleClose = () => {
+    setOpen(false)
+    showAgainProp = showAgain
+  }
+
+  useEffect(() => {
+    if (showAgainProp === true && open === false) {
+      setTimeout(() => {
+        setOpen(true)
+      }, 30000)
+    }
+  }, [showAgainProp])
```

Comment:
if I select to not show again, is it going to remember my choice forever or if I open in new tab it will pop up again?